tests/ci/integration/run_haproxy_integration.sh

```diff
@@ -25,10 +25,13 @@ export LD_LIBRARY_PATH="${AWS_LC_INSTALL_FOLDER}/lib"
 
 function build_and_test_haproxy() {
   cd ${HAPROXY_SRC}
-  make CC="${CC}" -j ${NUM_CPU_THREADS} TARGET=generic USE_OPENSSL=1 SSL_INC="${AWS_LC_INSTALL_FOLDER}/include" SSL_LIB="${AWS_LC_INSTALL_FOLDER}/lib/"
+  make CC="${CC}" -j ${NUM_CPU_THREADS} TARGET=generic USE_OPENSSL=1 SSL_INC="${AWS_LC_INSTALL_FOLDER}/include" \
+      SSL_LIB="${AWS_LC_INSTALL_FOLDER}/lib/" USE_LUA=1  LUA_LIB_NAME=lua5.4
 
-  # These tests pass when run locally but are not supported in CodeBuild, see CryptoAlg-1965
-  excluded_tests=("mcli_show_info.vtc" "mcli_start_progs.vtc" "tls_basic_sync.vtc" "tls_basic_sync_wo_stkt_backend.vtc" "acl_cli_spaces.vtc" "http_reuse_always.vtc")
+  # These tests pass when run locally but are not supported in CodeBuild, see CryptoAlg-1965.
+  # ssl_dh.vtc expects to use libssl with a FFDH ciphersuite which is unsupported, it will be gracefully turned off in
+  # ssl_dh.vtc with the change in https://github.com/andrewhop/haproxy/pull/1
+  excluded_tests=("mcli_show_info.vtc" "mcli_start_progs.vtc" "tls_basic_sync.vtc" "tls_basic_sync_wo_stkt_backend.vtc" "acl_cli_spaces.vtc" "http_reuse_always.vtc" "ocsp_auto_update.vtc" "ssl_dh.vtc")
```

Comment:
something wrong happened here ((

https://github.com/haproxy/haproxy/blob/master/reg-tests/peers/tls_basic_sync.vtc#L5

"tls_basic_sync.vtc" is marked as "SLOW", such tests are often not stable and should not be run regularly.

actually, supposed way of invocation is https://github.com/haproxy/haproxy/blob/master/.github/workflows/vtest.yml#L142


I consider that as lack of documentation. Probably we should add that convention somewhere
```
make reg-tests VTEST_PROGRAM=../vtest/vtest REGTESTS_TYPES=default,bug,devel
```

Reply:
This would work once https://github.com/andrewhop/haproxy/pull/1 is merged in which skips ssl_dh.vtc when HAProxy is built with AWS-LC. For now I have to keep the manual filter to skip ssl_dh.vtc.